flink-streaming-connectors/flink-connector-kafka/src/main/java/org/apache/flink/streaming/connectors/kafka/FlinkKafkaConsumer.java

```diff
@@ -567,6 +588,76 @@ public void notifyCheckpointComplete(long checkpointId) throws Exception {
 		}
 		return partitionsToSub;
 	}
+
+	/**
+	 * Thread to periodically commit the current read offset into Zookeeper.
+	 */
+	private static class PeriodicOffsetCommitter extends Thread {
+		private long commitInterval;
```

Comment:
(Option) I think it is good style to make the constant members final and sort them before the volatile/mutable ones. Helps code structure and readability...